src/System.Security.Cryptography.X509Certificates/tests/project.json

```diff
@@ -12,6 +12,7 @@
     "System.Runtime.InteropServices.RuntimeInformation": "4.3.0-beta-24522-03",
     "System.Security.Cryptography.Algorithms": "4.3.0-beta-24522-03",
     "System.Security.Cryptography.Cng": "4.3.0-beta-24522-03",
+    "System.Security.Cryptography.Csp": "4.3.0-beta-24522-03",
```

Comment:
I don't see anything in this change which should require adding a reference to CSP... what's this for?